docs/protocol.md

```diff
@@ -1,4 +1,7 @@
-# Overview
+# Protocol
+
+- [GitHub](https://github.com/dydxprotocol/solo)
+
 Solo is an open protocol consisting of smart contracts that run on the Ethereum blockchain. Solo supports margin trading, borrowing, and lending. The Solo Typescript library makes it easy to interact with the smart contracts running on the blockchain.
```

Comment:
The title says protocol, but this starts out "Solo.." which may be confusing for people who don't know what solo is